runelite-client/src/main/java/net/runelite/client/plugins/screenshot/ScreenshotPlugin.java

```diff
@@ -263,27 +263,31 @@ public void onGameTick(GameTick event)
 			return;
 		}
 
+		String screenshotSubDir = "";
```

Comment:
Set this to `null`, and null-check it along with fileName before calling `takeScreenshot()`.